completions/rsync

```diff
@@ -1,5 +1,36 @@
 # bash completion for rsync                                -*- shell-script -*-
 
+_rsync_vercomp() {
+    if [[ $1 == $2 ]]
+    then
+        return 0
+    fi
+    local IFS=.
+    local i ver1=($1) ver2=($2)
+    # fill empty fields in ver1 with zeros
+    for ((i=${#ver1[@]}; i<${#ver2[@]}; i++))
+    do
+        ver1[i]=0
+    done
+    for ((i=0; i<${#ver1[@]}; i++))
+    do
+        if [[ -z ${ver2[i]} ]]
```

Comment:
https://github.com/scop/bash-completion/blob/master/doc/styleguide.md#x-and--x-vs--n-x-and--z-x